launcher/internal/oci/interface.go

```diff
@@ -0,0 +1,59 @@
+// Package oci contains functionalities to interact with OCI image signatures.
+// https://github.com/opencontainers/image-spec/tree/main#readme.
+package oci
+
+import (
+	"go.uber.org/multierr"
+)
+
+// SigningAlgorithm is a specific type for string constants used for sigature signing and verification.
+type SigningAlgorithm string
+
+const (
+	// RsassaPssSha256 is RSASSA-PSS with a SHA256 digest supported for cosign sign.
+	RsassaPssSha256 SigningAlgorithm = "RSASSA_PSS_SHA256"
```

Comment:
The current implementations use Tink to verify signatures. Since Tink only accepts at least 2048 bit key, the key size here should be at least 2048 bit long. The `SigningAlgorithm` will be used by Tink to determine which signing algorithm to use for verification.

Reply:
It might be worth documenting this, since I would normally expect that key strength is specified as part of the signature algorithm.

Reply:
Whoops, nvm, saw that was added!